### PR TITLE
Explain approval rights

### PR DESCRIPTION
**Pull request by bkchr**
This also adds a tutorial on how to add the github name to the on-chain identity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,13 @@ Do not modify the `relay/` or `system-parachains/` folder. They are currently re
...
+
+## Approval rights
+
+The approval rights are configured in [`review-bot.yml`](.github/review-bot.yml). The rights are configured as:
+
+- All files in `.github` require two approvals from fellow members of rank 4 or higher.
+- `CHANGELOG.md`, `relay/*` or `system-parachains/*` require four approvals from fellow members of rank 3 or higher.
+- All other files require the approval from one fellow member of rank 2 or higher.
+
+The review-bot is using the on chain identity to map from a github account to a fellowship member. This requires that each fellowship member adds its github handle to its on-chain identity. Check [here](docs/github-on-chain-identity.md) for an instruction.
```
Comment (joepetrowski) [inline README.md:36]:
```suggestion
The review-bot uses the on-chain identity to map from a github account to a fellowship member. This requires that each fellowship member add their github handle to their on-chain identity. Check [here](docs/github-on-chain-identity.md) for instructions.
```

**Review by kianenigma — APPROVED** [review]
Thanks for the description! I will add github identity feild soon

File: .github/review-bot.yml [hunk 2]
```diff
@@ -9,24 +9,20 @@ rules:
...
     condition:
       include:
-        - ^relay\/kusama\/.*
-        - ^relay\/polkadot\/.*
+        - ^relay\/.*
         - ^system-parachains\/.*
-        - ^CHANGELOG$
-      exclude: 
-        - ^relay\/.+\.adoc$
+        - ^CHANGELOG.md$
```
Comment (Bullrich) [inline .github/review-bot.yml:14]: The `CHANGELOG` file does not have a file type.

```suggestion
        - ^CHANGELOG$
```

File: .github/review-bot.yml [hunk 3]
```diff
@@ -9,24 +9,20 @@ rules:
...
-      exclude: 
-        - ^relay\/kusama\/.*
-        - ^relay\/polkadot\/.*
-        - ^\.github/.*
+      exclude:
+        - ^relay\/.*
         - ^system-parachains\/.*
-        - ^target\/.*
+        - ^\.github/.*
+        - ^CHANGELOG.md$
```
Comment (Bullrich) [inline .github/review-bot.yml:26]: Also here. Btw, by excluding it here it won't be protected by the review bot as now it doesn't fit into any `include` rules.

```suggestion
        - ^CHANGELOG$
```

File: README.md [hunk 4]
```diff
@@ -24,3 +24,13 @@ Do not modify the `relay/` or `system-parachains/` folder. They are currently re
     └── collectives
         └── collectives-polkadot
 ```
+
+## Approval rights
+
+The approval rights are configured in [`review-bot.yml`](.github/review-bot.yml). The rights are configured as:
+
+- All files in `.github` require two approvals from fellow members of rank 4 or higher.
+- `CHANGELOG.md`, `relay/*` or `system-parachains/*` require four approvals from fellow members of rank 3 or higher.
```
Comment (Bullrich) [inline README.md:33]:
```suggestion
- `CHANGELOG`, `relay/*` or `system-parachains/*` require four approvals from fellow members of rank 3 or higher.
```

File: docs/github-on-chain-identity.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Adding github account to your on-chain identity
+
+Adding the github account to your identity requires the usage of the `additional` fields in the identity info. This is currently only supported by using the [polkadot-js bare extrinsic interface](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics). See the following image for an example:
```
Comment (kianenigma) [inline docs/github-on-chain-identity.md:3]: Might be worth emphasizing that the identities are set in the relay chain, while the fellowship itself lives in collectives?

File: .github/review-bot.yml (hunk 2 above)
Comment (bkchr) [reply .github/review-bot.yml:14]: https://github.com/polkadot-fellows/runtimes/pull/28/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed I changed it here

Comment (Bullrich) [reply .github/review-bot.yml:14]: Ah, I didn't see it. I was comparing it against `main`. 

Thanks for clarifying it!

File: .github/review-bot.yml (hunk 3 above)
Comment (bkchr) [reply .github/review-bot.yml:26]: It is part of the include rules for "Relay and system files"?

Comment (Bullrich) [reply .github/review-bot.yml:26]: Ah yes, you are right.

I got it confused with the way git showed the file changes and missread the `exclude` section.

File: docs/github-on-chain-identity.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+![Add github name to the additional fields of the on-chain identity](github-on-chain-identity-process.jpg)
+
+You should also at least add the `display` field. If you want to add `utf8` characters, you need to hex encode them and put the hex string (`0x` prefixed) into the field.
```
Comment (rzadp) [inline docs/github-on-chain-identity.md:7]: Might be worth to warn the user that leaving `None` values will override any information already existing on-chain.

```suggestion
You should also at least add the `display` field. If you want to add `utf8` characters, you need to hex encode them and put the hex string (`0x` prefixed) into the field.

If you have other identity fields (`display`, `legal`, `email`, etc) already configured on-chain, you have to repeat them in this extrinsic (`None` values will override the existing information).
```

[the remark this answers is not in the document]
Comment (joepetrowski) [reply docs/github-on-chain-identity.md:7]: `display` is typically what UIs use to render your name.

[the remark this answers is not in the document]
Comment (rzadp) [reply docs/github-on-chain-identity.md:7]: Ok, I thought maybe it's technically required for the `review-bot`.

[the remark this answers is not in the document]
Comment (joepetrowski) [reply docs/github-on-chain-identity.md:7]: That I do not know

**Review by xlc — APPROVED** [review]

**Review by ordian — APPROVED** [review]
I remember there was a requirement about adding the onchain address to github bio or something like that. Is it no longer required? Otherwise, would be good to document as well.

File: README.md [hunk 7]
```diff
@@ -24,3 +24,13 @@ Do not modify the `relay/` or `system-parachains/` folder. They are currently re
...
+- All other files require the approval from one fellow member of rank 2 or higher.
```
Comment (joepetrowski) [inline README.md:34]: Technically, a "Fellow" is _member of the Fellowship_ rank 3 or higher. "Rank 2 Fellow" doesn't make sense, they are "Members".
```suggestion
- All files in `.github` require two approvals from Fellowship members of rank 4 or higher.
- `CHANGELOG.md`, `relay/*` or `system-parachains/*` require four approvals from Fellowship members of rank 3 or higher.
- All other files require the approval from one Fellowship member of rank 2 or higher.
```

File: README.md [hunk 8]
```diff
@@ -24,3 +24,13 @@ Do not modify the `relay/` or `system-parachains/` folder. They are currently re
...
+
+The review-bot uses the on-chain identity to map from a github account to a fellowship member. This requires that each fellowship member add their github handle to their on-chain identity. Check [here](docs/github-on-chain-identity.md) for instructions.
```
Comment (joepetrowski) [inline README.md:36]:
```suggestion
The review-bot uses the on-chain identity to map from a GitHub account to a Fellowship member. This requires that each Fellowship member add their GitHub handle to their on-chain identity. Check [here](docs/github-on-chain-identity.md) for instructions.
```

**Review by joepetrowski — APPROVED** [review]

**Comment by bkchr** [comment]
> I remember there was a requirement about adding the onchain address to github bio or something like that. Is it no longer required? Otherwise, would be good to document as well.

That is not a requirement of any of this logic. This was only required for the initial seeding.

File: docs/github-on-chain-identity.md [hunk 9]
```diff
@@ -0,0 +1,9 @@
+# Adding github account to your on-chain identity
+
+Adding the github account to your identity requires the usage of the `additional` fields in the identity info. The identity needs to be set on Polkadot. This is currently only supported by using the [polkadot-js bare extrinsic interface](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.polkadot.io#/extrinsics/decode/0x1c010407676974687562123c67697468756220757365726e616d653e0000000000000000). See the following image for an example:
```
Comment (ordian) [inline docs/github-on-chain-identity.md:3]: maybe link a more comprehensive guide from @andresilva here: https://hackmd.io/@andresilva/rJ8fGMtf6 ?

**Review by andresilva — APPROVED** [review]

File: docs/github-on-chain-identity.md (hunk 9 above)
Comment (andresilva) [reply docs/github-on-chain-identity.md]: I just pushed it here now.

**Pull request merged**